src/main/java/com/bamdoliro/gati/domain/board/presentation/dto/request/CreateBoardRequestDto.java

```diff
@@ -18,16 +17,19 @@ public class CreateBoardRequestDto {
     @NotNull
     private Long communityId;
     @NotNull
+    @Size(min = 3, max = 20)
     private String title;
     @NotNull
+    @Size(min = 10, max = 4000)
     private String content;
 
     public Board toEntity(User writer, Community community) {
-        return Board.builder()
+        Board board = Board.builder()
                 .title(title)
                 .content(content)
                 .community(community)
                 .writer(writer)
                 .build();
+        return board;
```

Comment:
```suggestion
        return board = Board.builder()
                .title(title)
                .content(content)
                .community(community)
                .writer(writer)
                .build();
```
바로 return 해주면 될 거 같습니다!~